From 557bf1bd692a588867721c9ec94bb8e5002ea9f9 Mon Sep 17 00:00:00 2001
From: Vivek Mahajan <vivek.mahajan@freescale.com>
Date: Mon, 14 Dec 2009 11:31:38 +0530
Subject: [PATCH 1/1] net: 85xx: add more qoriq ids in PCIe host driver

Add new QorIQ ids in the PCIe host/agent driver

Signed-off-by: Vivek Mahajan <vivek.mahajan@freescale.com>
---
 drivers/net/fsl_pci_veth/pci_boardnet.c |   20 ++++++++++++++++++++
 1 files changed, 20 insertions(+), 0 deletions(-)

diff --git a/drivers/net/fsl_pci_veth/pci_boardnet.c b/drivers/net/fsl_pci_veth/pci_boardnet.c
index df82f38..b99bfca 100644
--- a/drivers/net/fsl_pci_veth/pci_boardnet.c
+++ b/drivers/net/fsl_pci_veth/pci_boardnet.c
@@ -688,6 +688,26 @@ static struct pci_device_id boardnet_id_table[] = {
 	 PCI_ANY_ID, PCI_ANY_ID, 0x0b2001, 0xffffff, 0},
 	{PCI_VENDOR_ID_FREESCALE, PCI_DEVICE_ID_P2020,
 	 PCI_ANY_ID, PCI_ANY_ID, 0x0b2001, 0xffffff, 0},
+	{PCI_VENDOR_ID_FREESCALE, PCI_DEVICE_ID_P2010E,
+	 PCI_ANY_ID, PCI_ANY_ID, 0x0b2001, 0xffffff, 0},
+	{PCI_VENDOR_ID_FREESCALE, PCI_DEVICE_ID_P2010,
+	 PCI_ANY_ID, PCI_ANY_ID, 0x0b2001, 0xffffff, 0},
+	{PCI_VENDOR_ID_FREESCALE, PCI_DEVICE_ID_P1011E,
+	 PCI_ANY_ID, PCI_ANY_ID, 0x0b2001, 0xffffff, 0},
+	{PCI_VENDOR_ID_FREESCALE, PCI_DEVICE_ID_P1011,
+	 PCI_ANY_ID, PCI_ANY_ID, 0x0b2001, 0xffffff, 0},
+	{PCI_VENDOR_ID_FREESCALE, PCI_DEVICE_ID_P1013E,
+	 PCI_ANY_ID, PCI_ANY_ID, 0x0b2001, 0xffffff, 0},
+	{PCI_VENDOR_ID_FREESCALE, PCI_DEVICE_ID_P1013,
+	 PCI_ANY_ID, PCI_ANY_ID, 0x0b2001, 0xffffff, 0},
+	{PCI_VENDOR_ID_FREESCALE, PCI_DEVICE_ID_P1020E,
+	 PCI_ANY_ID, PCI_ANY_ID, 0x0b2001, 0xffffff, 0},
+	{PCI_VENDOR_ID_FREESCALE, PCI_DEVICE_ID_P1020,
+	 PCI_ANY_ID, PCI_ANY_ID, 0x0b2001, 0xffffff, 0},
+	{PCI_VENDOR_ID_FREESCALE, PCI_DEVICE_ID_P1022E,
+	 PCI_ANY_ID, PCI_ANY_ID, 0x0b2001, 0xffffff, 0},
+	{PCI_VENDOR_ID_FREESCALE, PCI_DEVICE_ID_P1022,
+	 PCI_ANY_ID, PCI_ANY_ID, 0x0b2001, 0xffffff, 0},
 	{ }
 };
 MODULE_DEVICE_TABLE(pci, boardnet_id_table);
-- 
1.5.6.5

